doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -108,6 +108,44 @@ For each empty box in the matrix above, ensure that the following applications c
    gnome-terminal
    ```
 
+5. Test that `X11` apps can be started and can receive input when `XWayland` is supported:
+    ```sh
+    sudo apt install x11-apps
+    
+    # first,
+    xeyes
+
+    # then
+    xedit
+
+    # finally
+    xcalc
+    ```
+
+6. (X11) Test that `mattermost` can be opened and resized. This was first encountered in
+   [#2509](https://github.com/canonical/mir/issues/2509). To test:
```

Comment:
I think this was a longstanding bug, not a 2.16 regression.

I don't doubt it needs testing, but this sort of arithmetic error should probably have a unit test.